apps/mobile/src/features/ConfirmTx/ConfirmTx.container.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { ScrollView, View } from 'tamagui'
+import { useScrollableHeader } from '@/src/navigation/useScrollableHeader'
+import { NavBarTitle } from '@/src/components/Title'
+import { TransactionInfo } from './components/TransactionInfo'
+import {
+  MultisigExecutionDetails,
+  useTransactionsGetTransactionByIdV1Query,
+} from '@safe-global/store/gateway/AUTO_GENERATED/transactions'
+import { useDefinedActiveSafe } from '@/src/store/hooks/activeSafe'
+import { RouteProp, useRoute } from '@react-navigation/native'
+import { ConfirmationView } from './components/ConfirmationView'
+import { LoadingTx } from './components/LoadingTx'
+import { useTxSigner } from './hooks/useTxSigner'
+import { Alert } from '@/src/components/Alert'
+import { ConfirmTxForm } from './components/ConfirmTxForm'
+
+function ConfirmTxContainer() {
+  const activeSafe = useDefinedActiveSafe()
+  const txId = useRoute<RouteProp<{ params: { txId: string } }>>().params.txId
+
+  const { handleScroll } = useScrollableHeader({
+    children: <NavBarTitle paddingRight={5}>Confirm transaction</NavBarTitle>,
+    alwaysVisible: true,
+  })
+
+  const { data, isFetching, isError } = useTransactionsGetTransactionByIdV1Query({
+    chainId: activeSafe.chainId,
+    id: txId,
+  })
+
+  const detailedExecutionInfo = data?.detailedExecutionInfo as MultisigExecutionDetails
+  const { activeSigner, hasSigned } = useTxSigner(detailedExecutionInfo)
+  const hasEnoughConfirmations =
+    detailedExecutionInfo?.confirmationsRequired === detailedExecutionInfo?.confirmations?.length
```

Comment:
```suggestion
    detailedExecutionInfo?.confirmationsRequired <= detailedExecutionInfo?.confirmations?.length
```

There can be more signatures than needed, our backend allows that.

Reply:
Yes, but in the app we need to change the "confirm" button in case the user has enough signatures. We show this layout here: https://www.figma.com/design/yVshG47sZRM7ztI66bDII4/New-Mobile-App?node-id=5000-25641&t=4NQx81UKm8wXZ3Bm-0 , so the user will see that he can only execute it in the web app, that is why I check for enough confirmations.

Reply:
What I'm saying is, that a tx can have MORE confirmations than needed.